basics/the-blockchain/actors.md

```diff
@@ -1,133 +1,141 @@
+The `InitActor` initializes new actors on the Filecoin network. It maps public keys and temporary actor addresses to canonical ID addresses. This actor is instantiated in the genesis block and interacts directly with the FVM.
 
 #### AccountActor
 
-The `AccountActor` is responsible for user accounts. Account actors are not created by the `InitActor` but by sending a message to a public-key style address. The account actor updates the state tree with a new actor address and interacts directly with the FVM.
+The `AccountActor` manages user accounts by updating the state tree with new actor addresses. It interacts directly with the FVM.
 
 #### RewardActor
 
-The `RewardActor` manages unminted Filecoin tokens and distributes rewards directly to miner actors, where they are locked for vesting. The reward value used for the current epoch is updated at the end of an epoch. The `RewardActor` interacts directly with the FVM.
+The `RewardActor` distributes rewards to miner actors based on their contributions. Rewards are locked for vesting, and values are updated at the end of each epoch. This actor interacts directly with the FVM.
 
 #### StorageMarketActor
 
-The `StorageMarketActor` is responsible for processing and managing on-chain deals. This is also the entry point of all storage deals and data into the system. This actor keeps track of storage deals and the locked balances of both the client storing data and the storage provider. When a deal is posted on-chain through the `StorageMarketActor`, the actor will first check if both transacting parties have sufficient balances locked up and include the deal on-chain. Additionally, the `StorageMarketActor` holds _Storage Deal Collateral_ provided by the storage provider to collateralize deals. This collateral is returned to the storage provider when all deals in the sector successfully conclude. This actor does not interact directly with the FVM.
+The `StorageMarketActor` processes and manages on-chain deals. It tracks storage deals and the locked balances of both clients and storage providers. When a deal is posted, it ensures both parties have sufficient collateral before including the deal on-chain. Collateral is returned when deals are successfully completed. This actor does not interact directly with the FVM.
 
 #### StorageMinerActor
 
-The `StorageMinerActor` is created by the `StoragePowerActor` and is responsible for storage mining operations and the collection of mining proofs. This actor is a key part of the Filecoin storage mining subsystem, which ensures a storage miner can effectively commit storage to Filecoin and handles the following:
+The `StorageMinerActor` manages storage mining operations and proof collection. It ensures miners can effectively commit storage and handles:
```

Comment:
```suggestion
The `StorageMinerActor` manages storage provider operations and proof collection. It ensures storage providers can effectively commit storage and handles:
```
The actor code still contains the term Miner but it should be changed to StorageProviderActor at some point in line with [FIP0018](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0018.md).